cypress/integration/DropdownMenu.spec.ts

```diff
@@ -0,0 +1,280 @@
+describe('DropdownMenu', () => {
+  describe('given submenu user', () => {
+    beforeEach(() => {
+      cy.visitStory('dropdownmenu--submenus');
+    });
+
+    describe('when using pointer', () => {
+      beforeEach(() => {
+        cy.findByText('Open').click();
+      });
+
+      it('should open submenu and not focus first item when moving pointer over trigger', () => {
+        pointerOver('Bookmarks →')
+          .then(getAssociatedSubmenu)
+          .children()
+          .first()
+          .should('not.be.focused');
+      });
+
+      it('should not close when moving pointer to submenu and back to parent trigger', () => {
+        pointerOver('Bookmarks →');
+        pointerOver('Inbox');
+        pointerOver('Bookmarks →');
+        cy.findByText('Inbox').should('be.visible');
+      });
+
+      it(
+        'should close submenu when moving pointer away but remain open when moving towards',
+        {
+          viewportWidth: 550,
+        },
+        () => {
+          // Moving away
+          pointerOver('Bookmarks →');
+          cy.findByText('Inbox').should('be.visible');
+          pointerExitRightToLeft('Bookmarks →');
+          cy.findByText('Inbox').should('not.exist');
+
+          // Moving towards
+          pointerOver('Bookmarks →');
+          cy.findByText('Inbox').should('be.visible');
```

Comment:
This looks redundant but what's really happening is we're leaning on the retry-ability of the assertion to cover the 100ms timeout on the open, the alternative is to chain a `wait(100)` but that couples us closer to the implementation.

You'll notice this in a few places, perhaps there's a better way to present it.